web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -72,6 +73,13 @@
                 HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(SampleListNotFoundException.class)
+    public ResponseEntity<ErrorResponse> handleCaseListNotFound(SampleListNotFoundException ex) {
```

Comment:
should this function name have been "handleSampleListNotFoundException" ?  (sorry .. just noticed this now)